sandpack-react/src/components/CodeEditor/CodeMirror.stories.tsx

```diff
@@ -123,13 +123,13 @@ export const CustomLanguagePython: React.FC = () => (
     <CodeEditor
       additionalLanguages={[
         {
-          name: "python",
+          name: "python" as const, // TODO: how to convert it to a union type instead of a string?
```

Comment:
This is a cool approach! Does it need to be a union, since it would never be concerned about existing file types?